### DISCUSSION TITLE
Redisinsight

### DISCUSSION THREAD

**Discussion by hughdidit** [docker/extension-ideas#184] [Ideas]
That's it. Just need redisinsight in Docker.

**Comment by ajeetraina** [comment]
Did you check https://github.com/collabnix/redisinsight-docker-extension

**Comment by hughdidit** [comment]
No.

Thanks I will.


Hugh Chapman
www.hughdidit.com
(510) 497-1957

*Be initiated! Join the mailing list. Get a free comic.
<http://eepurl.com/gjzCZ1>*

FOLLOW ME:
***@***.***_art <http://instagram.com/hughdidit_art>
***@***.*** <http://twitter.com/hughdidit>
***@***.*** <http://fb.com/hughdidit>

*Fan Rewards On Patreon! <http://www.patreon.com/hughdidit> *
